### ISSUE TITLE
[Discussion] Spreading cLHS generated sample points

### ISSUE THREAD

**Issue by Samuel22** [soiltechproject/soiltech-tools#3]
Some feedback from FarmLab, the cLHS points are bunched. When sampling multiple paddocks, this can result in samples being contained within a single paddock. Suggest two 'options' for the user:
- Add Lat/Lon as a dimension to better spread the points
- Include the paddock as a dimension to spread across individual paddocks (when running cLHS across a farm)

**Comment by KipCrossing** [comment]
Good point. The current python script runs all the columns by default and, via the CLI, we exclude the lat/lon columns. In the short term, for FarmLab, we could simply not exclude these columns.

Looking forward, however, this won't be so obvious from the scala interfaces perspective @correllink . This could be done like:

```scala
val samplesLocations = samsFarm.GenSampleLocations("clhc", 10, zones = [paddock01, paddock02], attributes = ["ndvi", "uGamma"], includeLatLon = true,  includeZones = true)
```

Remember that `paddock01` and `paddock02` would be zones with `type="paddock"`

---

In the back end, for the paddock zones, do we simply number them and use that column *OR* do we implement [Objective Factor 2](https://github.com/soiltechproject/soiltech/blob/master/src/main/python/lhc/METHOD.md) that handles categorical data?
